db/schema.rb

```diff
@@ -49,6 +49,7 @@
     t.datetime "updated_at"
     t.datetime "applied_at"
     t.integer  "updater_id"
+    t.text     "state",                       default: "draft", null: false
```

Comment:
Also, in my experience, `varchar` offers no gains over `text` in PostgreSQL, but `text` has the upside that you'll never have to recast the column in case you need something more than e.g. 255 chars.